doc/getting_started/index.rst

```diff
@@ -29,85 +29,55 @@ Follow one of the following guides for your host operating system.
 Get the source code
 *******************
 
-Zephyr's multi-purpose :ref:`west` tool lets you easily get the Zephyr project
-source code, instead of manually cloning the Zephyr repos along with west
-itself.
-
-.. warning::
-
-   It's possible to use Zephyr without installing west, but you have
-   to **really** know :ref:`what you are doing <no-west>`.
+Zephyr's multi-purpose :ref:`west <west>` tool lets you easily get the Zephyr
+project repositories. (While it's easiest to develop with Zephyr using west,
+we also have :ref:`documentation for doing without it <no-west>`.)
 
 Bootstrap west
 ==============
 
-First, install the ``west`` binary and bootstrapper:
+First, install the ``west`` binary and bootstrapper using ``pip3``:
 
 .. code-block:: console
 
    # Linux
    pip3 install --user west
 
-   # macOS and Windows
+   # macOS (Terminal) and Windows (cmd.exe)
    pip3 install west
 
-.. note::
-   See :ref:`west-install` for additional details on installing west.
+See :ref:`west-install` for additional details on installing west. See
+`Installing Packages`_ in the Python Packaging User Guide for more information
+about pip\ [#pip]_, including this `information on -\\-user`_.
 
-.. _clone-zephyr:
+- On Linux, make sure ``~/.local/bin`` is on your :envvar:`PATH`
+  :ref:`environment variable <env_vars>`, or programs installed with ``--user``
+  -- like west -- won't be found\ [#linux_user]_.
 
-Clone the Zephyr Repositories
-=============================
+- On macOS, `Homebrew disables -\\-user`_.
 
-.. warning::
-   If you have run ``source zephyr-env.sh`` (on Linux or macOS) or
-   ``zephyr-env.cmd`` (on Windows) on a clone of zephyr that predates the
-   introduction of west, then the copy of west included in the clone will
-   override the bootstrapper installed with ``pip``. In that case close the
-   shell and open a new one in order to remove it from the ``PATH``. You
-   can check which ``west`` is being executed by running::
+- On Windows, see the Installing Packages information on ``--user`` if you
+  require using this option.
 
-      west --version
+.. _clone-zephyr::
```

Comment:
Whoops, only one colon at the end of the line (that's generating the malformed label error)